src/core/agent_user_manager.py

```
@@ -384,10 +384,21 @@ async def ensure_core_users_exist(self):
 
         await self.user_manager.ensure_core_users_exist(core_users)
 
-    async def sync_agents_to_users(self):
-        """Main function to sync Letta agents to Matrix users"""
-        logger.info("Starting agent-to-user sync process")
-        print("[AGENT_SYNC] Starting agent-to-user sync process", flush=True)
+    async def sync_agents_to_users(self):
+        """Main function to sync Letta agents to Matrix users"""
+        import time
```

Ruff (0.14.5)

Redefinition of unused `time` from line 9

Remove definition: `time`

(F811)